src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServicePermissionTests.cs

```diff
@@ -179,7 +180,7 @@ public async void GetMyReleasesForReleaseStatusesAsync_CanViewAllReleases()
             
             var service = new ReleaseService(contentDbContext.Object, AdminMapper(), 
```

Comment:
Seems like we could refactor out a `createReleaseService` method for all of these tests.

Reply:
Done